packages/server/src/modules/transactions/providers/transactions.provider.ts

```diff
@@ -149,6 +149,23 @@ const getTransactionsByFilters = sql<IGetTransactionsByFiltersQuery>`
   ORDER BY event_date DESC;
 `;
 
+interface IGetTransactionsWithNoDocumentQuery {
+  params: {
+    ownerId: string;
+  };
+  result: IGetTransactionsByFiltersResult;
+}
+
+const getTransactionsWithChargeButNoDocument = sql<IGetTransactionsWithNoDocumentQuery>`
+  SELECT t.*
+  FROM accounter_schema.extended_transactions t
+  LEFT JOIN accounter_schema.documents d ON t.charge_id = d.charge_id
+  WHERE t.charge_id IS NOT NULL 
+    AND d.id IS NULL
+    AND t.owner_id = $ownerId
+  ORDER BY event_date DESC;
+`;
```

Comment:
This query might miss on those cases:
If a charge has an invoice, and user uploads a receipt
If charge has proforma or statement doc, and user uploads invoice/receipt
If a charge is constructed of multiple invoices

While those are not the common charge, it worth having some solution for them as well.
Maybe we could have a flow for the common "easy" cases, and if no results found fall to a more robust flow